spec/ios_lint_localizations_spec.rb

```diff
@@ -37,7 +37,7 @@
             fi
           SCRIPT
           FileUtils.mkdir_p File.join(install_dir, 'bin')
-          File.write(File.join(install_dir, 'bin/swiftgen'), script, perm: 0766)
+          File.write(File.join(install_dir, 'bin/swiftgen'), script, perm: 0o766)
```

Comment:
Is there a format that can do like `0x766` or something? 

Not a huge deal, but I feel like if I look at this in six months I'm gonna be like "what _is_ that string?"

But then I'mm probably just look at the `File.write` call and figure it out in like 2 seconds, so not a huge deal – more just curious as it seems unusual.

Reply:
> Is there a format that can do like 0x766 or something?

`0x` is the prefix for hexadecimal, while `0o` is the prefix for octal, which is what is used with `chmod`-like permissions. So `0o` is that format that can do it :P

Note: I looked at https://ruby-doc.org/core-2.5.0/File/Constants.html to see if there were constants we could bitwise-or to provide the same value in a more readable way, but those seems to be focused on providing constants for things like the open modes (read/read-write/append, text/binary) not on the Unix permissions.

Agree that since `0o` is not as widely used/known as `0x` it might need a quick comment though.

```suggestion
          # note: `0o` is octal notation, used to specify chmod-like flags
          File.write(File.join(install_dir, 'bin/swiftgen'), script, perm: 0o766)
```